### DISCUSSION TITLE
Add "Submit To All" button on the Offer File sharing screen in GUI

### DISCUSSION THREAD

**Discussion by OverActiveBladderSystem** [Chia-Network/chia-blockchain-gui#2371] [Ideas]
I imagine some users would want to share any offer file they create to every site possible for listing.

Rather than select one by one, a "submit to all" button allowing them one-click submission to all the sites.
![Screenshot (1661)](https://github.com/Chia-Network/chia-blockchain/assets/36348012/c8a000fa-f1a0-4777-84d6-a51bca86bc20)

**Comment by BrandtH22** [comment]
@OverActiveBladderSystem , note I transferred this enhancement request to the GUI repo for better visibility 👍
